.github/dependabot.yml

```
@@ -0,0 +1,7 @@
+version: 2
+updates:
+  # Maintain dependencies for GitHub Actions
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```

let's do monthly, it was proven in several different repos that weekly is a bit too often and feels spammy

```suggestion
      interval: "monthly"
```